sandbox.go

```diff
@@ -104,6 +104,7 @@ type resolvConfPathConfig struct {
 type containerConfig struct {
 	hostsPathConfig
 	resolvConfPathConfig
+	name              string
```

Comment:
If we don't foresee a good reason to have the name I will remove it. @mrjana WDYT ?